src/ssl.c

```diff
@@ -33577,9 +33577,31 @@ int wolfSSL_DH_set0_pqg(WOLFSSL_DH *dh, WOLFSSL_BIGNUM *p,
 
     return WOLFSSL_SUCCESS;
 }
+
 #endif /* v1.1.0 or later */
 #endif /* !HAVE_FIPS || HAVE_FIPS_VERSION > 2 */
 
+/**
+ * retrive p, q and g parameter
```

Comment:
Spelling - "retrive" should be "Retrieve"

Reply:
By rebasing those comments are have been fixed.